### PR TITLE
Define the "prefix_lists" model for the Dell Enterprise SONiC collection

### PR DESCRIPTION
**Pull request by kerry-meyer**
Initial posting for the prefix_lists model definition.

### CODE REVIEW

File: models/enterprise_sonic/prefix_lists/sonic_prefix_lists.yml [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  options:
+    config:
+      description:
+        - Specifies a list of prefix set configuration dictionaries
+      type: list
+      elements: dict
+      suboptions:
+        name:
+          description:
+            - name of a prefix set (a list of prefix entries)
```
Comment (stalabi1) [inline models/enterprise_sonic/prefix_lists/sonic_prefix_lists.yml:31]: Maybe you'd want to capitalize this line for consistency.

Comment (kerry-meyer) [reply models/enterprise_sonic/prefix_lists/sonic_prefix_lists.yml]: Thanks for the reminder. I have capitalized that line.

**Comment by stalabi1** [comment]
I approve of these changes.

**Review by stalabi1 — APPROVED** [review]
I approve of these changes.

**Pull request merged**